src/core_plugins/i18n/server/tool/i18n_verify_keys.js

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs';
```

Comment:
So to verify that the translations exist *and* they're configured properly, I think we're going to have to boot up a Kibana server instance, ask the i18n module for all the registered translations for the default locale, and check them against all the translation keys extracted from any Jade templates found in the source. We could still do this during the build process, similar to [how the optimize step is run](https://github.com/elastic/kibana/blob/7323b95d5be4ba26a0494bff981eb96619bf1e92/tasks/build/index.js#L22-L22).

@epixa does that sound reasonable to you?